drivers/adc/ad7606/ad7606.h

```diff
@@ -232,8 +243,8 @@ struct ad7606_range {
 	int32_t min;
 	/** Maximum range value */
 	int32_t max;
-	/** Whether the range is differential */
-	bool differential;
+	/** Type of range according to @enum ad7606_range_type */
```

Comment:
seems like doxygen is complaining about how you reference here the range_type enum. Please use the doxygen commands for this:
https://www.doxygen.nl/manual/commands.html#cmdref